jujugui/static/gui/src/app/app.js

```diff
@@ -437,7 +437,7 @@ YUI.add('juju-gui', function(Y) {
             '#user-name .header-banner__link--breadcrumb');
         var auth = this._getAuth('charmstore');
         if (breadcrumbElement) {
-          breadcrumbElement.textContent = auth && auth.user && auth.user.name ||
+          breadcrumbElement.textContent = auth && auth.user ||
```

Comment:
Yeah this will conflict with my branch but that's ok because all of this will be removed. Are you sure that this is a good path even for JEM?